app/grandchallenge/evaluation/views/__init__.py

```diff
@@ -1412,3 +1419,56 @@ def get_context_data(self, *args, **kwargs):
             }
         )
         return context
+
+
+class PhaseStarterKitInfo(
+    LoginRequiredMixin,
+    ObjectPermissionRequiredMixin,
+    DetailView,
+):
+    model = Phase
+    permission_required = "evaluation.change_phase"
+    raise_exception = True
+    template_name = "evaluation/phase_starter_kit.html"
+
+    def get_object(self, queryset=None):
+        return get_object_or_404(
+            Phase,
+            challenge=self.request.challenge,
+            slug=self.kwargs["slug"],
+        )
+
+
+class PhaseStarterKitDownload(
+    LoginRequiredMixin,
+    ObjectPermissionRequiredMixin,
+    DetailView,
+):
+    model = Phase
+    permission_required = "evaluation.change_phase"
+    raise_exception = True
+
+    def get(self, *_, **__):
+        phase = self.get_object()
```

Comment:
Don't you need a custom `get_object` here as well to subset by challenge?

Reply:
You could probably subclass the `PhaseStartKitDetail` view here, since the permissions are the same as well.